minibot/bot.py

```diff
@@ -2,7 +2,7 @@
 Minibot object.
 """
 import multiprocessing
-from multiprocessing import Queue
+from queue import Queue
```

Comment:
lol

Reply:
yes? whats wrong here?

Reply:
nothing just that previously we imported Queue from multiprocessing and not queue LOL

Reply:
now that you mention it why were we even importing queue from multiprocessing

Reply:
ahhh i cant merge when this comment thread still exists